app/models/tag_user.rb

```diff
@@ -259,6 +259,7 @@ def self.notification_levels_for(user)
 #
 # Indexes
 #
-#  idx_tag_users_ix1  (user_id,tag_id,notification_level) UNIQUE
-#  idx_tag_users_ix2  (tag_id,user_id,notification_level) UNIQUE
+#  idx_tag_users_ix1                      (user_id,tag_id,notification_level) UNIQUE
```

Comment:
In my opinion, these two indexes do not make sense to be `UNIQUE` and should be recreated, but I do not think there is any overhead so maybe we can just leave them.